popout.js

```diff
@@ -143,20 +145,21 @@ class PopoutModule {
 				// Add delay before rendering in case some things aren't done initializing, like sheet templates
 				// which get loaded asynchronously.
 				Hooks.on('ready', () => setTimeout(() => PopoutModule.renderPopout(${sheet}), 1000));
-		  	window.dispatchEvent(new Event('load'))
-		  </script>`))
+		  	window.dispatchEvent(new Event('load'));
+		  </script>`));
 		// Open new window and write the new html document into it
 		// We need to open it to the same url because some images use relative paths
 		let windowFeatures = undefined;
 		if (game.settings.get("popout", "useWindows"))
-			windowFeatures = 'toolbar=0,location=0,menubar=0,titlebar=0,scrollbars=1';
-		let win = window.open(window.location.toString(), '_blank', windowFeatures)
+			windowFeatures = 'toolbar=0, location=0, menubar=0, titlebar=0, scrollbars=1, width=800, height=900';
+		let win = window.open(window.location.toString(), '_blank', windowFeatures);
 		//console.log(win)
 		// Need to specify DOCTYPE so the browser is in standards mode (fixes TinyMCE and CSS)
-		win.document.write("<!DOCTYPE html>" +  html[0].outerHTML)
+		win.document.write("<!DOCTYPE html>" +  html[0].outerHTML);
 		// After doing a write, we need to do a document.close() so it finishes
 		// loading and emits the load event.
-		win.document.close()
+		win.document.close();
+		obj.sheet.close;

```

Comment:
I would put the `obj.sheet.close` behind a `game.settings.get('popout', 'closeSheet')` and have the setting registered on the `ready` hook, so users can enable/disable this behavior. I'd think it makes sense for the default to be `true` though in this case.